mkdocs.yml

```diff
@@ -12,6 +12,10 @@ pages:
     - Home: 'networking/index.md'
     - Overview: 'networking/overview.md'
     - SimpleImpl: 'networking/simpleimpl.md'
+  - Data Storage:
+    - Capabilities: 'datastorage/capabilities.md'
```

Comment:
Not sure if this is the best category. Maybe a new category such as "Forge Systems"?